.golangci.yml

```diff
@@ -14,7 +14,6 @@ issues:
         - lll
     - path: "kardinal/*"
       linters:
-        - dupl
```

Comment:
Re-enabled this linter now that those apply resources functions are not duplicated.